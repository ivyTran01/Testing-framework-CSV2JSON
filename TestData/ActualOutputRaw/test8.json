[
{"field1":"","field2":22,"field3":true,"field4":5.2}
]
