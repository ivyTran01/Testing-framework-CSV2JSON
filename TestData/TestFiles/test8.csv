"",22,true,5.2
