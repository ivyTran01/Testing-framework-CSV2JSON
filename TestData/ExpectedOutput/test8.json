[
 {
  "field1":null,
  "field2":22,
  "field3":true,
  "field4":5.2
 }
]